dts/bindings/sensor/vishay,vcnl4040.yaml

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) 2020 Richard Osterloh
+# SPDX-License-Identifier: Apache-2.0
+
+description: VCNL4040 proximity and ambient light sensor
+
+compatible: "vishay,vcnl4040"
+
+include: i2c-device.yaml
+
+properties:
+    int-gpios:
+      type: phandle-array
+      required: false
+      description: |
+        The INT pin signals that a programmable interrupt function
+        for ALS and PS with upper and lower thresholds has been
+        triggered. The sensor generates an active-low level signal
+        which remains asserted until the data is read.
+
+    led-current:
+      type: string
```

Comment:
Is there a reason these integer-valued properties shouldn't be type `int`?  enum does work with non-string types.

Reply:
Changed to int and retested. Still works as expected. Out of interest what difference does this make if it's still an enum?

Reply:
Forgot to answer this: the macros generated for property values can be affected by the type that the values represent.  A string zero might be generated as `"0"` which is not useful when it's used as an integer.  (It may be that it has no effect in the current tooling, but the type should still represent how the value is expected to be used.)